### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception  java.lang.NullPointerException: Cannot invoke "com.microsoft.azure.toolkit.lib.common.task.AzureTaskManager.runOnPooledThread(java.lang.Runnable)" because the return value of "com.microsoft.azure.toolkit.lib.common.task.AzureTaskManager.getInstance()" is null

### ISSUE THREAD

**Issue by florentbo** [microsoft/azure-tools-for-java#6939]
IntelliJ build version: 2022.2.1 IU-222.3739.54
OS: Linux
JDK: JetBrains s.r.o. 17.0.3
Plugin version: 3.68.1-2022.2
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,0,37,1848x1016,invalid,layout=com.intellij.openapi.wm.impl.IdeRootPane$MyRootLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=null, throwable=java.lang.NullPointerException: Cannot invoke "com.microsoft.azure.toolkit.lib.common.task.AzureTaskManager.runOnPooledThread(java.lang.Runnable)" because the return value of "com.microsoft.azure.toolkit.lib.common.task.AzureTaskManager.getInstance()" is null
	at com.microsoft.azure.toolkit.intellij.cosmos.dbtools.DbToolsWorkaround.preload(DbToolsWorkaround.java:41)
```

**Comment by wangmingliang-ms** [comment]
thanks for reporting, this bug should be fixed by commit [d691dcc](https://github.com/microsoft/azure-tools-for-java/commit/d691dcc3e63f9b294232a0e79220a5034e6a8e14), and the fix would be included in the next stable release.

**Comment by wangmingliang-ms** [comment]
close as it has been fixed since v3.69.0, you can upgrade to the latest and try. Feel free to reopen it when you encounter this error again.
